src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -1007,12 +1007,13 @@ case_ = unsafeSqlCase
 --   Primary bar
 -- @
 --
--- For this example, declare:
+-- In this example, Bar is said to be the BaseEnt(ity), and Foo the child.
+-- To model this in Esqueleto, declare:
 --
 -- @
 -- instance ToBaseId Foo where
 --   type BaseEnt Foo = Bar
---   toBaseIdWitness = FooKey
+--   toBaseIdWitness barId = FooKey barId
```

Comment:
Lev and I thought removing the eta reduction makes it more clear what this function is doing